### DISCUSSION TITLE
Daterangepicker disable

### DISCUSSION THREAD

**Discussion by RaulFerreiraSilva** [ant-design/ant-design#41799] [General]
How can I disable the second date of a daterangepicker without having to pass a default value or that it returns today?

**Comment by Yuiai01** [comment]
You might try the following
![image](https://user-images.githubusercontent.com/112228030/231909338-497394eb-53c4-400b-89a1-a660ab4efb40.png)

**Reply by RaulFerreiraSilva** [reply]
I've tried, but this returns today as the default value of the input.

**Reply by Yuiai01** [reply]
> I've tried, but this returns today as the default value of the input.

If you are using antd 5.x then you can set `RangePicker` value to `[date, null]`.

**Reply by RaulFerreiraSilva** [reply]
I am going to try

**Reply by RaulFerreiraSilva** [reply]
![image](https://user-images.githubusercontent.com/105366476/232128262-113b4430-9efc-4961-b072-c65f49efeb47.png)

Continue with the same behavior, the second disabled input brings today's date with default value
